LayoutTests/fast/repaint/border-radius-repaint-2-expected.txt

```diff
@@ -0,0 +1,7 @@
+layer at (0,0) size 800x600
```

Comment:
This doesn't look like the correct output for a repaint test.

Reply:
Sorry, I haven't noticed that repaint.js was deprecated by text-based-repaint.js.

I've updated it to use text-based and added a platform/iOS specific -expected file, since the rects are slightly different on iOS. Is this expected? @smfr